File: Kattis/CPP/IntroProblems/r2.cc
/*
kattis R2 problem
Author: Agis Daniels
Solves for r2 in mean calc (r1+r2)/2 == s
NOTES
*/
#include <bits/stdc++.h>

using namespace std;

int main (){
    int r=0,s=0;
    cin>>r>>s;
    cout<<((r>s)? s-(r-s): s+(s-r))<<endl;
    return 0;
}

File: Kattis/Python/IntroProblems/r2.py
""" 
kattis R2 problem
Author: Agis Daniels
Solves for r2 in mean calc (r1+r2)//2 == s
"""
from sys import stdin as rf
def main():
    r,s=map(int, rf.readline().split())
    ans= s-(r-s) if r>s else s+(s-r)
    print("{}".format(ans))
if __name__=='__main__':
    main()
